src/hooks/screen/useChatScreenHook.ts

```diff
@@ -147,7 +195,7 @@ function useChatScreenHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
       if (e?.response?.data?.status === 'Channel is blocked') return;
 
       setTimeout(() => {
-        sendChat(message, iteration + 1, currentChatSchema).catch((sendChatError) => {
+        sendChat(message, attachments, iteration + 1, currentChatSchema).catch((sendChatError) => {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `sendChat` function is now expecting an `attachments` parameter, but it's not provided in the recursive call within the `setTimeout`. This could lead to unexpected behavior if the initial call to `sendChat` included attachments.
```diff
- sendChat(message, attachments, iteration + 1, currentChatSchema).catch((sendChatError) => {
+ sendChat(message, iteration + 1, currentChatSchema, attachments).catch((sendChatError) => {
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->